components/engine_rocks/Cargo.toml

```diff
@@ -48,8 +49,9 @@ protobuf = "2"
 fail = "0.4"
 
 [dependencies.rocksdb]
-git = "https://github.com/tikv/rust-rocksdb.git"
+git = "https://github.com/tabokie/rust-rocksdb.git"
```

Comment:
please change it

Reply:
This PR currently depends on tikv/rust-rocksdb#595